source/includes/steps-general-connection-tab.yaml

```diff
@@ -0,0 +1,66 @@
+---
+title: Click on :guilabel:`Advanced Connection Options`.
```

Comment:
Just curious, did you try the new rst step syntax for these steps? I see you've used them further down in this file, so wondering if we might be able to use them here as well to avoid yaml.

https://github.com/terakilobyte/docs-node/blob/foo/source/usage-examples/find.txt#L92-L117

Reply:
Done.